web/build.gradle

```diff
@@ -36,7 +36,7 @@ dependencies {
     compile ([group: 'org.apache.httpcomponents', name: 'httpasyncclient', version: '4.1.3'])
     provided 'com.opensymphony:xwork:2.0.4' // Struts 2
     provided 'org.springframework:spring-webmvc:3.1.0.RELEASE'
-    provided group: 'javax.servlet', name: 'servlet-api', version: '2.5'
+    provided group: 'javax.servlet', name: 'javax.servlet-api', version: '3.0.1'
```

Comment:
Please make sure to register this on OSS before merging.

Reply:
Pending review